tox.ini

```diff
@@ -13,9 +13,7 @@ setenv=
 skip_install = true
 envdir = {toxworkdir}/py39
 commands=
-    ; Force packaging even if setup.{py,cfg} haven't changed
-    rm -rf ./*.egg-info/
-    python setup.py coverage
+    python setup.py coverage {posargs}
```

Comment:
same as the [comment on pathfinder](https://github.com/kytos-ng/pathfinder/pull/22#discussion_r896873551), wdyt about using python3 here to make it uniform with `setup.py`?

Reply:
done here [5040e24](https://github.com/kytos-ng/of_core/pull/69/commits/5040e242ec35aa0feecea524475950aedf4ca97b)